aws-rds-cfn-common/src/main/java/software/amazon/rds/common/logging/RequestData.java

```diff
@@ -1,23 +1,21 @@
 package software.amazon.rds.common.logging;
 
-import org.json.JSONObject;
+import java.util.Map;
 
+import com.google.common.collect.Maps;
 import lombok.AllArgsConstructor;
 import lombok.Data;
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
 
 @Data
 @AllArgsConstructor
 class RequestData {
-    private String stackId;
-    private String awsAccountId;
-    private String clientRequestToken;
+    private final Map<String, String> requestDataMap = Maps.newHashMap();
```

Comment:
I believe the initial purpose of this class was somewhat a data class. By now it is a thin wrapper for a hashmap. If this is our purpose, let's simply pass the hashmap around with no extra wrapping: the class is too shallow.

Reply:
Done